bundles/org.openhab.binding.worxlandroid/README.md

```diff
@@ -0,0 +1,649 @@
+
+| Channel  | Type        | ChannelName      |
+|----------|-------------|------------------|
+| enable   | Switch      | friday2#enable   |
+| time     | DateTime    | friday2#time     |
+| duration | Number:Time | friday2#duration |
+| edgecut  | Switch      | friday2#edgecut  |
+
+##### saturday (Slot 1)
+
+| Channel  | Type        | ChannelName       |
+|----------|-------------|-------------------|
+| enable   | Switch      | saturday#enable   |
+| time     | DateTime    | saturday#time     |
+| duration | Number:Time | saturday#duration |
+| edgecut  | Switch      | saturday#edgecut  |
+
+
+##### saturday2 (Slot 2, ONLY IF SUPPORTED)
+
+| Channel  | Type        | ChannelName        |
+|----------|-------------|--------------------|
+| enable   | Switch      | saturday2#enable   |
+| time     | DateTime    | saturday2#time     |
+| duration | Number:Time | saturday2#duration |
+| edgecut  | Switch      | saturday2#edgecut  |
+
+
+##### one-time
+
+| Channel  | Type   | ChannelName       |
+|----------|--------|-------------------|
+| edgecut  | Switch | one-time#edgecut  |
+| duration | Switch | one-time#duration |
+
+
+##### battery
+
+| Channel             | Type                     | ChannelName                 |
+|---------------------|--------------------------|-----------------------------|
+| temperature         | Number:Temperature       | battery#temperature         |
+| voltage             | Number:ElectricPotential | battery#voltage             |
+| level               | Number                   | battery#level               |
+| charge-cycles       | Number                   | battery#charge-cycles       |
+| charge-cycles-total | Number                   | battery#charge-cycles-total |
+| charging            | Switch                   | battery#charging            |
+
+
+##### orientation
+
+| Channel | Type         | ChannelName       |
+|---------|--------------|-------------------|
+| pitch   | Number:Angle | orientation#pitch |
+| roll    | Number:Angle | orientation#roll  |
+| yaw     | Number:Angle | orientation#yaw   |
+
+
+##### metrics
+
+| Channel          | Type          | ChannelName              |
+|------------------|---------------|--------------------------|
+| blade-time       | Number:Time   | metrics#blade-time       |
+| blade-time-total | Number:Time   | metrics#blade-time-total |
+| distance         | Number:Length | metrics#distance         |
+| total-time       | Number:Time   | metrics#total-time       |
+
+
+##### rain (ONLY IF SUPPORTED)
+
+| Channel | Type        | ChannelName  |
+|---------|-------------|--------------|
+| state   | Switch      | rain#state   |
+| counter | Number:Time | rain#counter |
+| delay   | Number:Time | rain#delay   |
+
+
+##### wifi
+
+| Channel      | Type         | ChannelName       |
+|--------------|--------------|-------------------|
+| rssi         | Number:Power | wifi#rssi         |
+| wifi-quality | Number       | wifi#wifi-quality |
+
+
+## Examples
+
```

Comment:
Please add the bridge/mower thing declaration too.

Reply:
Added